src/Form/Extension/Field/Type/FormTypeFieldExtension.php

```diff
@@ -145,9 +143,6 @@ public function buildView(FormView $view, FormInterface $form, array $options):
         } else {
             $view->vars['sonata_admin_enabled'] = false;
         }
-
-        // NEXT_MAJOR: Remove this line
-        $view->vars['sonata_admin'] = $sonataAdmin;
```

Comment:
I forgot to remove the NEXT_MAJOR comment in https://github.com/sonata-project/SonataAdminBundle/pull/5928/files#diff-c3866c555c015246d9d140a0e828f011L155, in this case is just to remove the comment and leave `$view->vars['sonata_admin'] = $sonataAdmin;`.

Reply:
Wow, I will take a look, probably will need to restore some test code I had to remove because of this line

Reply:
Fixed.